statistics/histogram.go

```diff
@@ -1609,6 +1626,13 @@ func (idx *Index) expBackoffEstimation(sctx sessionctx.Context, coll *HistColl,
 	return singleColumnEstResults[0] * math.Sqrt(singleColumnEstResults[1]) * math.Sqrt(math.Sqrt(singleColumnEstResults[2])) * math.Sqrt(math.Sqrt(math.Sqrt(singleColumnEstResults[3]))), true, nil
 }
 
+func (idx *Index) checkStats() {
+	if idx.IsEssentialStatsLoaded() {
+		return
+	}
+	HistogramNeededItems.insert(model.TableItemID{TableID: idx.PhysicalID, ID: idx.Info.ID, IsIndex: true})
```

Comment:
When we use index stats, we add it to `HistogramNeededItems` when topn is evicted while we don't add it when cmsketch is evicted?

Reply:
updated, we use isFullload instead.